tfhe/src/core_crypto/gpu/algorithms/test/lwe_programmable_bootstrapping_128.rs

```diff
@@ -78,7 +78,7 @@ where
 
     let gpu_index = 0;
     let stream = CudaStreams::new_single_gpu(GpuIndex::new(gpu_index));
-    let d_bsk = CudaLweBootstrapKey::from_lwe_bootstrap_key(&std_bootstrapping_key, &stream);
+    let d_bsk = CudaLweBootstrapKey::from_lwe_bootstrap_key(&std_bootstrapping_key, None, &stream);
```

Comment:
Have you executed the noise/pfail tests for PBS128 with drift @guillermo-oyarzun?